.neuro/live-s3.yml

```diff
@@ -75,22 +28,20 @@ jobs:
     pass_config: True
```

Comment:
Let's put this job def config (except of bash lines) from here and pahy live to project.live as mixin and use here, adding the bash section only.

Same for label studio, jupyter.
WDYT?